sdk/communication/communication-sms/test/utils/recordedClient.ts

```diff
@@ -33,18 +32,14 @@ export const recorderConfiguration: RecorderEnvironmentSetup = {
   queryParametersToSkip: []
 };
 
-export function createCredential(): TokenCredential | undefined {
-  if (isNode && isPlaybackMode()) {
+export function createCredential(): TokenCredential {
+  if (isPlaybackMode()) {
```

Comment:
Why is this change safe now? Running this in browser mode will throw, right?

Reply:
Yes it will, but I think we should not be running this in browser mode, and we can do so by changing our test structure by putting tests that should only run in node under the node folder.